packages/components/NativeRadioButton/macos/RadioButtonViewManager.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+@objc(RadioButtonViewManager)
+class RadioButtonViewManager: RCTViewManager {
```

Comment:
I think we should use a 3 letter prefix here https://microsoft.github.io/objc-guide/Headers/Naming.html

I used MSF in my PR for MenuButton, but there's a comment asking If that's the right one.

Reply:
I removed the prefix because it's not a Fluent control, though I agree with you that we should use a 3 letter prefix.

Reply:
Talked offline, seems like you'll follow up on this issue https://office.visualstudio.com/APEX/_workitems/edit/5319675